merlin-server/sql/merlin/tables/activity_directive.sql

```diff
@@ -13,15 +13,21 @@ create table activity_directive (
   last_modified_arguments_at timestamptz not null default now(),
   metadata merlin_activity_directive_metadata_set default '{}'::jsonb,
 
+  anchor_ID integer default null,
```

Comment:
Minor naming convention comment - we tend to keep everything lowercase

Reply:
Ah, un-quoted identifiers are case-insensitive, right? So this is more of a style comment than a naming one :+1:

Reply:
Either way, thanks for the catch. It is fixed now.